edgedb/asyncio_con.py

```diff
@@ -578,7 +578,7 @@ async def async_connect_raw(dsn: str = None, *,
                             user: str = None, password: str = None,
                             database: str = None,
                             tls_ca_file: str = None,
-                            tls_security: bool = None,
+                            tls_security: str = None,
```

Comment:
@jaclarke should the default be `'default'` or `None`?

Reply:
This is just IMO, but I think `None` is appropriate here. All default values for connection parameters (such as tls_security, hostname, database, etc.) are defined in the properties of `edgedb.con_utils.ResolvedConnectConfig` when they are accessed

Reply:
Yes, it should be `None`. `'default'` would override any value given in the `EDGEDB_CLIENT_TLS_SECURITY` env var, or the dsn and credentials_file options (and their env vars).